src/power_grid_model/_core/power_grid_model_c/get_pgm_dll_path.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+import os
+import platform
+from importlib.resources import files
+from pathlib import Path
+
+
+def get_pgm_dll_path() -> Path:
+    """
+    Returns the path to PGM dynamic library.
+    """
+    package_dir = Path(str(files(__package__)))
+    platform_name = platform.uname().system
```

Comment:
```suggestion
    platform_name = platform.uname().sysname
```

Reply:
ohhh apparently, you need to use `os.uname()`, not `platform.uname()`: https://docs.python.org/3/library/os.html#os.uname

see the documentation of https://docs.python.org/3/library/platform.html#platform.system which explicitly says you have to use `os.uname().sysname`